managed/Makefile

```diff
@@ -54,7 +54,7 @@ test-cover:                     ## Run tests and collect per-package coverage in
 	go test $(PMM_TEST_FLAGS) -p 1 -race -coverprofile=cover.out -covermode=atomic -coverpkg=./... ./...
 
 test-update:                   ## Run pmm update test
-	PMM_TEST_RUN_UPDATE=1 go test -timeout=180s -v -p 1 -race ./services/supervisord -run ^TestDevContainer/Update$$
+	PMM_TEST_RUN_UPDATE=1 go test -timeout=600s -v -p 1 -race ./services/supervisord -run ^TestDevContainer/Update$$
```

Comment:
Increased timeout here, because these tests failing by timeout if timeout is 3 minutes only.